### PR TITLE
A few updates for the documentation

### PR DESCRIPTION
**Pull request by lucianomartin**
Closes https://xmosjira.atlassian.net/browse/AP-210

This PR includes @shuchitak's commit 5dbf030344a95cd7badf719d29e6b8605453b24d to complete Jenkins' run.

### CODE REVIEW

File: doc/programming_guide/ffva/deploying/native_windows.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ Run the following commands in the root folder to build the USB firmware:
 
     cmake -G "NMake Makefiles" -B build -DCMAKE_TOOLCHAIN_FILE=xmos_cmake_toolchain/xs3a.cmake
     cd build
-    nmake example_ffva_ua_adec
+    nmake example_ffva_ua_adec_altarch
```
Comment (mbanth) [inline doc/programming_guide/ffva/deploying/native_windows.rst:51]: If you're updating the documentation, then please also change it from using nmake to ninja, and be sure to test that the instructions actually work with ninja.

**Review by mbanth — CHANGES_REQUESTED** [review]

File: doc/programming_guide/ffva/deploying/native_windows.rst (hunk 1 above)
Comment (lucianomartin) [reply doc/programming_guide/ffva/deploying/native_windows.rst]: Good point. I made the changes and I replaced `-DCMAKE_TOOLCHAIN_FILE` with `--toolchain` too.

File: doc/programming_guide/low_power_ffd/deploying/native_windows.rst [hunk 2]
```diff
@@ -5,10 +5,20 @@
 Deploying the Firmware with Native Windows
 ******************************************
 
-This document explains how to deploy the software using `CMake` and `NMake`. If you are not using
+This document explains how to deploy the software using *CMake* and *NMake*. If you are not using
```
Comment (mbanth) [inline doc/programming_guide/low_power_ffd/deploying/native_windows.rst:8]: Please replace `NMake` with `Ninja`

**Review by mbanth — CHANGES_REQUESTED** [review]

File: doc/programming_guide/low_power_ffd/deploying/native_windows.rst (hunk 2 above)
Comment (lucianomartin) [reply doc/programming_guide/low_power_ffd/deploying/native_windows.rst]: Fixed.

**Review by mbanth — APPROVED** [review]
All good now.

**Pull request merged**